src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -3128,6 +3342,10 @@ int32_t NaClSysClockGetCommon(struct NaClAppThread  *natp,
   }
 
  done:
+  #ifdef TRACING
+  NaClStraceClockGetCommon(nap->cage_id, clk_id, ts_addr, (uintptr_t)time_func, retval);
```

Comment:
does this function need a trace call?

Reply:
it seems like this is not an actual syscall, and this function is not even in the .h file, so it's a helper function, we don't want to trace this one